admin/src/main/resources/io/buoyant/admin/js/src/router_client.js

```diff
@@ -99,10 +103,37 @@ define([
       return summary;
     }
 
-    return function (metricsCollector, routers, client, $metricsEl, routerName, $chartEl, colors, $toggleLinks, shouldExpandInitially) {
+    function barChartColorFn(percent) {
+      return percent < 0.5 ? "orange" : "green";
+    }
+
+    function barPercentCalc(data) {
+      if (!data) return null;
+
+      var numer = data["loadbalancer/available"] || {};
+      var denom = data["loadbalancer/size"] || {};
+      var percent = (!denom || !denom.value) ? 0 : (numer.value || 0) / denom.value;
+
+      return {
+        percent: percent,
+        label: {
+          description: "Endpoints available",
+          value: (numer.value || "-") + " / " + (denom.value || "-")
+        }
+      }
+    }
+
+    return function (metricsCollector, routers, client, $container, routerName, colors, shouldExpandInitially) {
       var metricPartial = Handlebars.compile(metricPartialTemplate);
       Handlebars.registerPartial('metricPartial', metricPartial);
 
+      var $contentContainer = $container.find(".client-content-container");
+
+      var $metricsEl = $container.find(".metrics-container");
+      var $chartEl = $container.find(".chart-container");
+      var $toggleLinks = $container.find(".client-toggle");
```

Comment:
I refactored these - they were being passed in originally:
```
return function (metricsCollector, routers, client, $metricsEl, routerName, $chartEl, colors, $toggleLinks, shouldExpandInitially)
```

They're used later on in the function - maybe github has cut the rest of the function off:

```
    return function (metricsCollector, routers, client, $container, routerName, colors, shouldExpandInitially) {
      var metricPartial = Handlebars.compile(metricPartialTemplate);
      Handlebars.registerPartial('metricPartial', metricPartial);

      var $contentContainer = $container.find(".client-content-container");

      var $metricsEl = $container.find(".metrics-container");
      var $chartEl = $container.find(".chart-container");
      var $toggleLinks = $container.find(".client-toggle");
      var $lbBarChart = $container.find(".lb-bar-chart");

      var clientColor = colors.color;
      var latencyLegend = createLatencyLegend(colors.colorFamily);
      var metricDefinitions = getMetricDefinitions(routerName, client.label);

      var $expandLink = $toggleLinks.find(".client-expand");
      var $collapseLink = $toggleLinks.find(".client-collapse");

      renderMetrics($metricsEl, client, [], [], clientColor);
      var chart = SuccessRateGraph($chartEl.find(".client-success-rate"), colors.color);
```